internal/component/artifactory.go

```diff
@@ -68,7 +69,13 @@ func NewArtifactoryReleaseSource(c cargo.ReleaseSourceConfig) *ArtifactoryReleas
 }
 
 func (ars *ArtifactoryReleaseSource) DownloadRelease(releaseDir string, remoteRelease cargo.BOSHReleaseTarballLock) (Local, error) {
-	downloadURL := ars.ArtifactoryHost + "/artifactory/" + ars.Repo + "/" + remoteRelease.RemotePath
+	u, err := url.Parse(ars.ArtifactoryHost)
```

Comment:
It's missing an error check here.

Reply:
Thanks, Chris.

Reply:
Added a test to check for malformed URLs and the error check itself. `test.sh` is passing locally.